app/views/layouts/application.html.haml

```diff
@@ -0,0 +1,36 @@
+!!!
+%html
+  %head
+    %meta{:charset => "utf-8"}/
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace